app/aimodels/bertopic/routers/bertopic_embedding_pretrained.py

```diff
@@ -0,0 +1,79 @@
+import os
+from typing import Union
+from fastapi import Depends, APIRouter, UploadFile
+from pydantic import UUID4
+from ..schemas.bertopic_embedding_pretrained import BertopicEmbeddingPretrained, BertopicEmbeddingPretrainedCreate, BertopicEmbeddingPretrainedUpdate
+from app.dependencies import get_db, get_minio
+from sqlalchemy.orm import Session
+from .. import crud
+from ..ai_services.basic_inference import BASE_CKPT_DIR
+from ..models.bertopic_embedding_pretrained import BertopicEmbeddingPretrainedModel
+from app.core.errors import HTTPValidationError, ValidationError
+from aiofiles import open as open_aio
+from minio import Minio
+
+
+router = APIRouter(
+    prefix="/bertopic-embedding-pretrained"
+)
+
+
+@router.post(
```

Comment:
created story to pickle, save, and return sha256 for sentence_models prior to upload